test/FunctionalTests/Microsoft.OData.Edm.Tests/Validation/ValidationRuleSetTests.cs

```diff
@@ -22,7 +22,9 @@ public class ValidationRuleSetTests
         public void NewValidationRulesShouldBeInTheRuleSetExceptBaselinedExceptionRules()
         {
             var validationRules = new Dictionary<object, string>();
-            var items = typeof(ValidationRules).GetFields().Where(f => f.Name != "NavigationPropertyEntityMustNotIndirectlyContainItself")
+            var items = typeof(ValidationRules).GetFields().Where(f =>
+                f.Name != "NavigationPropertyEntityMustNotIndirectlyContainItself" &&
+                f.Name != "EntityTypeKeyMissingOnEntityType")

```

Comment:
>"EntityTypeKeyMissingOnEntityType" [](start = 26, length = 34)

I'd like to keep it unremoved and add the key in the corresponding test cases?

Reply:
We can't remove it, because it is public, but we don't want it to be part of the ValidationRules.  This test verifies that all of the defined tests are part of the validation rules, so we need to omit the two that are defined but should not be part of the validation rules.

---
In reply to: [174973472](https://github.com/OData/odata.net/pull/1109#discussion_r174973472) [](ancestors = 174973472)